hosts

```diff
@@ -7,11 +7,13 @@
 
 #devapi ansible_host=devapi.opentreeoflife.org ansible_user=ubuntu ansible_ssh_private_key_file=/home/ejmctavish/.ssh/opentree.pem
 
-otindexdev ansible_host=otindexdev.opentreeoflife.org ansible_user=admin ansible_ssh_private_key_file=/home/ejmctavish/.ssh/opentree.pem
-
+##devotindex ansible_host=devotindex.opentreeoflife.org ansible_user=admin ansible_ssh_private_key_file=/Users/jima/.ssh/id_rsa
+##devotindex ansible_host=devotindex.opentreeoflife.org ansible_user=admin ansible_ssh_private_key_file=/Users/jima/.ssh/opentree/production.pem
+devotindex ansible_host=devotindex.opentreeoflife.org ansible_user=ubuntu ansible_ssh_private_key_file=/Users/jima/.ssh/opentree/production.pem
 
 [devapi]
-devapi ansible_host=devapi.opentreeoflife.org ansible_user=admin ansible_ssh_private_key_file=/home/ejmctavish/.ssh/opentree.pem
+#devapi ansible_host=devapi.opentreeoflife.org ansible_user=admin ansible_ssh_private_key_file=/Users/jima/.ssh/id_rsa
+devapi ansible_host=devapi.opentreeoflife.org ansible_user=admin ansible_ssh_private_key_file=/Users/jima/.ssh/opentree/production.pem
```

Comment:
why are we putting the key in a specific user (jima, ejmctavish) dir rather than a generic user?

Reply:
I think this was changed as as a marker that we weren't using a shared key anymore, and were each using our own?

Reply:
I'm happy to try and formalize an approach, if we want to go back to shared keys. Honestly, I just kept trying different auth methods until I found a user+key that would connect to ot55. 😬